src/main/resources/application-local.yaml

```
@@ -1,60 +1,63 @@
-  # 한국관광공사_관광지 집중률 방문자 추이 예측 정보
-  tats-cnctr-rate-service:
-    service-key: ${TATS_CNCTR_RATE_SERVICE_KEY}
-  data-lab-service:
-    service-key: ${DATA_LAB_SERVICE}
-
-etc-api:
-  # 기상청 (Korea Meteorological Administration)
-  kma:
-    vilage-fcst-info-service:
-      service-key: ${VILAGE_FCST_INFO_SERVICE_KEY}
+spring:
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)